aiohasupervisor/client.py

```diff
@@ -146,7 +155,7 @@ async def get(
         self,
         uri: str,
         *,
-        params: dict[str, str] | None = None,
+        params: dict[str, str] | MultiDict[str, str] | None = None,
```

Comment:
Seems `Query` would need a newer aiohttp version, it came in with [this PR](https://github.com/aio-libs/aiohttp/pull/9081) in 3.11.0 I think (currently pyproject.toml specifies 3.3.0 as minimum).

Let's go with the fixed `MultiDict` typing for now.

```suggestion
        params: dict[str, str] | MultiDict[str] | None = None,
```